packages/components/src/presentation-components/properties/inputs/AsyncSelect.tsx

```diff
@@ -122,6 +122,9 @@ export function AsyncSelect<OptionType, Group extends GroupBase<OptionType>, Add
   const { portalTarget } = usePortalTargetContext();
   const divRef = useRef<HTMLDivElement>(null);
 
+  // Wrap custom menu as a workaround for some internal bugs of react-select
+  const MenuList = wrapMenuList(CustomMenuList);
```

Comment:
Can you move with outside AsyncSelect to avoid wrapping on each render?

Reply:
Moved.